transactions/views.py

```diff
@@ -416,7 +416,7 @@ def foreign_card_validation_response(request):
 @swagger_auto_schema(method='post', request_body=CardlessPaymentSerializer)
 @api_view(['POST'])
 @permission_classes((IsAuthenticated, ))
-def tokenized_card_payment(request):
+def tokenized_card_payment(request, **kwargs):
     """
     Endpoint for handling payment using tokenized cards. The user makes
     payment without providing card details
```

Comment:
What do you think of changing this docstring, since it's not correct anymore. A user now has to provide a card id as part of the request.

Reply:
Yes @Tevinthuku, I corrected it.